### PR TITLE
Documentation and example for JMeter Sampler with MongoDB

### PR DESCRIPTION
**Pull request by eliflores**
And fix a linting issue.

**Note**:  This is related to the changes in the [kangal-jmeter](https://github.com/hellofresh/kangal-jmeter) image, PR https://github.com/hellofresh/kangal-jmeter/pull/28.

### CODE REVIEW

File: docs/jmeter/writing-tests.md [hunk 1]
```diff
@@ -409,3 +409,57 @@ For the full list of possible environment variables check [Kangal environment va
...
+
+    return String.format("Document with id %s deleted", result.get("_id"))
+}
+catch (Exception e) {
+    SampleResult.setSuccessful(false)
+    SampleResult.setResponseCode("500")
+    SampleResult.setResponseMessage("Something went wrong: " + e)
+}
+```
+7. Save the test plan as a `jmx` file. You can the location of the file to Kangal via the `TEST_PATH` environment variable.
```
Comment (s-radyuk) [inline docs/jmeter/writing-tests.md:465]: I'm a bit confused by this TEST_PATH env. It's not the variable from Kangal configuration. Where is it coming from?

**Comment by s-radyuk** [comment]
I think, with the latest release we should update this [doc](https://github.com/hellofresh/kangal/blob/master/docs/jmeter/README.md?plain=1#L12) too because users should create their jmx tests with the same [version of JMeter](https://github.com/hellofresh/kangal/blob/master/docs/jmeter/README.md?plain=1#L27) that Kangal uses.

File: docs/jmeter/writing-tests.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (eliflores) [reply docs/jmeter/writing-tests.md]: Yes, indeed! Good guess 😄

Comment (eliflores) [reply docs/jmeter/writing-tests.md]: Great point 😄  - this is coming from the way I was testing the changes of the [kangal-jmeter](https://github.com/hellofresh/kangal-jmeter) and this example.
I updated the phrase in [22f4993](https://github.com/hellofresh/kangal/pull/247/commits/22f4993d45918de008af1a7ef35652c171da3e3e). 
Thank you for the catch!

**Review by s-radyuk — APPROVED** [review]

**Pull request merged**